### ISSUE TITLE
push 0.2.0 to jcenter

### ISSUE THREAD

**Issue by almibe** [almibe/CodeEditor#1]

**Comment by almibe** [comment]
https://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/

https://github.com/bintray/gradle-bintray-plugin

**Comment by agusmba** [comment]
Hi almibe,

I was following your project because in the past I played around with JavaFX.
I saw your comments on this issue and wanted to let you know that I recently (past month) went through the process of uploading to maven central and it wasn't too complicated.

I even did it with gradle, which I believe you are using also.  See [my build file](https://bitbucket.org/smree/eemws-core/src/master/build.gradle?fileviewer=file-view-default)

Let me know if you want some pointers. I didn't try the bintray way so I cannot tell you which one'll be easier ;-)

Cheers!
